src/rules/imports-first.js

```diff
@@ -1,5 +1,12 @@
 import first from './first'
 
-const newMeta = Object.assign({}, first.meta, { deprecated: true })
+const ruleDocsUrl = 'https://github.com/benmosher/eslint-plugin-import/blob'
+
+const newMeta = Object.assign({}, first.meta, {
+  deprecated: true,
+  docs: {
+    url: `${ruleDocsUrl}/7b25c1cb95ee18acc1531002fd343e1e6031f9ed/docs/rules/imports-first.md`,
```

Comment:
i'm confused, why does this one have a sha but the others don't? could it be an optional second argument passed to `docsUrl`?

Reply:
The rule was deprecated / renamed in https://github.com/benmosher/eslint-plugin-import/commit/8e35cd93322efc478f205056833a17cd039e1902, however the documentation was directly renamed, meaning the straight mapping used elsewhere can't be used here.

An alternative here would be to simply point this rule at `first`'s documentation, as outlined in the PR description.

Reply:
Can we modify `docsUrl` to take an optional second argument that defaults to `master` but can be overridden to this sha?

Reply:
The URL's are slightly different, but sure I can fix it to that.

Reply:
Gave the function an optional second parameter, added tests for it, and used it for `imports-first`.